### DISCUSSION TITLE
Welcome to okbinder Discussions! 👋

### DISCUSSION THREAD

**Discussion by 7hens** [7hens/okbinder#8] [Q&A]
大家好，欢迎提出你的疑问

**Comment by TangStudio** [comment]
[TestApplication.zip](https://github.com/7hens/okbinder/files/8349078/TestApplication.zip)
[ServiceDemo.zip](https://github.com/7hens/okbinder/files/8349079/ServiceDemo.zip)
麻烦帮看看这2个工程，根据你的工程拆分出来成了2个APP，但是服务端没收到客户端发来的信息

**Reply by 7hens** [reply]
好的，你稍等

**Reply by 7hens** [reply]
初步发现以下问题：

1. TestApplication 应该服务端，但是在它的 Manifest 文件里面没有注册 `RemoteService`
2. ServiceDemo 应该是客户端，它的 Manifest 文件中`<queries>` 里面注册的服务端的包名不对
3. TestApplication 的 MainActivity 中，因为没有给 logView 赋值，导致它的 `print()` 方法直接报空指针异常

把上述问题修改过来后，我测试通过了
